minecraft-server/start-minecraftFinalSetup

```diff
@@ -54,8 +54,12 @@ done
 
 if [ "$TYPE" = "SPIGOT" ]; then
   if [ -d /plugins ]; then
+    # Copy all of the plugins directory first to include initial configs
     echo Copying any Bukkit plugins over
     cp -r /plugins /data
```

Comment:
Shouldn't this `cp` be removed then?

Reply:
Yes you are right. I added copying the mods with rsync as well.